### PR TITLE
fix: allow refund_policy to be none

### PR DESCRIPTION
**Pull request by snitin315**
for https://github.com/fossasia/open-event-frontend/issues/5036

### CODE REVIEW

File: app/models/event.py [hunk 1]
```diff
@@ -119,9 +119,7 @@ class Event(SoftDeletionModel):
     ical_url = db.Column(db.String)
     xcal_url = db.Column(db.String)
     is_sponsors_enabled = db.Column(db.Boolean, default=False)
-    refund_policy = db.Column(
-        db.String, default='All sales are final. No refunds shall be issued in any case.'
-    )
+    refund_policy = db.Column(db.String)
```
Comment (snitin315) [inline app/models/event.py:122]: we need a migration for past events, right?

**Comment by iamareebjamal** [comment]
Yes, we need to set refund policy to null for previous events.

**Comment by iamareebjamal** [comment]
Thinking about it now, if some user changed their refund policy using the API, making it null will lead to data loss. So, wait for Mario to tell what should be done

**Comment by snitin315** [comment]
> Thinking about it now, if some user changed their refund policy using the API, making it null will lead to data loss. So, wait for Mario to tell what should be done

I think we can apply migration only to events having the default `refund_policy` message.

**Comment by mariobehling** [comment]
In regards to events we should delete the policy. There was not any organizer who edited the policy, we just introduced it a few hours ago.

Is that easy to do? I dont think we need migrations.

**Comment by iamareebjamal** [comment]
We didn't add it a few hours ago. We used it as default refund policy for all events in the backend for a long time ago

**Pull request merged**